frontend/src/components/App/Layout.tsx

```diff
@@ -178,11 +183,13 @@ export default function Layout({}: LayoutProps) {
         <TopBar />
         <Sidebar />
         <main id="main" className={classes.content}>
-          {clusters && !!clusterInURL && !Object.keys(clusters).includes(getCluster() || '') ? (
-            <ClusterNotFoundPopup />
-          ) : (
-            ''
-          )}
+          {
+            // Check if none of the clusters in the URL are known.
+            clusters &&
+              clusterGroup.length > 0 &&
+              clusterGroup.filter(cluster => Object.keys(clusters).includes(cluster)).length ===
+                0 && <ClusterNotFoundPopup />
```

Comment:
I think this still needs the '' part? Because otherwise things on the left of '&&' could be shown?